deployment/terraform.tfvars.example

```diff
@@ -6,6 +6,3 @@ influxdb_bucket = "trento-telemetry"
 
 # ECS deployment data
 name = "trento-telemetry"
-region = "eu-west-2"
```

Comment:
I wouldn't remove the `region` here as it is still usable, and something that most of the users touch

Reply:
Good call, I have added back the variables with default values as commented items.